src/email_bridge.rs

```diff
@@ -93,7 +90,8 @@ pub fn request(app: &Config, email_addr: EmailAddress, client_id: &str, nonce: &
 pub fn verify(app: &Config, stored: &HashMap<String, String>, code: &str)
               -> BrokerResult<(String, String)> {
 
-    if code != &stored["code"] {
+    let trimmed = code.to_lowercase().replace(|c| !CODE_CHARS.contains(&(c as u8)), "");
```

Comment:
Should we be stripping out unrecognized characters? If we ever remove a character from `CODE_CHARS`, this has the side effect of invalidating unused codes generated by the previous version of the broker.

Not a problem for real people until we hit 1.0, but still... I'm at worst a -0 on this, but if an obvious, easy way to only strip whitespace occurs to you, let's do it.